app/payment-link-v2/amount/amount.njk

```diff
@@ -23,25 +23,19 @@
       }
     }) }}
 
-    <form action="/pay/{{ productExternalId }}/amount" method="post" name="startPaymentForm" class="push-bottom">
+    <form method="post" name="startPaymentForm" class="push-bottom">
```

Comment:
Could probably do without the name, as "startPaymentForm" doesn't really apply any more. But can be removed in a separate PR.

Reply:
Good idea - have removed it.

Reply:
Also removed the `class` attribute as it is not required.